src/core/stream/index.ts

```diff
@@ -398,14 +402,39 @@ export default function Stream({
       return Observable.of(evt);
     });
 
-    /**
-     * Add management of events linked to live Playback.
-     * @type {Observable}
+    /* In the case where minimumUpdatePeriod is specified, and above 0,
```

Comment:
You must put at least two asterisks to indicate that it is JSDOC.
Then a newline.
And you also forgot the returns JSDOC